connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverterConfig.java

```diff
@@ -131,4 +144,12 @@ public boolean replaceNullWithDefault() {
         return replaceNullWithDefault;
     }
 
+    /**
+     * 
```

Comment:
```suggestion
     * If a default schema is provided in the converter config, this will be
     * used for all messages.
     * 
     * This is only relevant if schemas are enabled.
     *
```